main.py

```diff
@@ -0,0 +1,47 @@
+import telebot
+import pyqrcode
+from PIL import Image
+from pyzbar.pyzbar import decode
+
+bot = telebot.TeleBot('6046831990:AAGMjVSRxnL_WX9IdfVcMLgjyoUcIjmSBYg')
```

Comment:
You should move this token outside of the repo. Everyone can take it and obtain access to your bot.

You can store it in environment variables.